pkg/manager/manager.go

```
@@ -786,3 +973,322 @@
+		}
+
+		// Check if this rule already exists
+		key := sr.Key
+		if existingRules[key] {
+			// Determine what to do based on autoResolve
+			action := autoResolve
+			if action == "" {
+				// Ask user what to do
+				fmt.Printf("Rule '%s' already exists. [s]kip, [o]verwrite, [r]ename? ", sr.Key)
+				var input string
+				fmt.Scanln(&input)
```

golangci-lint (1.64.8)

Error return value of `fmt.Scanln` is not checked

(errcheck)